build.fsx

```diff
@@ -19,35 +27,58 @@ let sublimePath () =
     [  Environment.GetEnvironmentVariable("SUBLIME_TEXT_DATA")
        Environment.ExpandEnvironmentVariables(@"%APPDATA%\Sublime Text 3") ]
 
-  let isWindows = (Path.DirectorySeparatorChar = '\\')
   let searchPaths = if isWindows then WindowsPaths else UnixPaths
   let directories =
      searchPaths
      |> List.filter Directory.Exists
 
   match directories.Length with
-     | 0 ->
-         trace "No Sublime text 3 installation found"
+  | 0 -> trace "No Sublime text 3 installation found"
```

Comment:
I usually go for 2 or 3. There's only one .fsx file so no biggie. :)
I'll go ahead and align on 4 in this PR.